### PR TITLE
feat: add rk3568-gpu/hdmi-disable.dts

### PR DESCRIPTION
**Pull request by CodeChenL**
Meets the low-power requirements of some users

### CODE REVIEW

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]
disable-gpu.dts
另外ci应该会报错：makefile没有这两个文件

File: arch/arm64/boot/dts/rockchip/overlays/rk3568-hdmi-disable.dts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/dts-v1/;
+/plugin/;
+
+/ {
+	metadata {
+		title = "Disable HDMI";
+		compatible = "rockchip,rk3568", "rockchip,rk3566";
+		category = "misc";
+		exclusive = "hdmi", "hdmi_sound";
+		description = "Disable HDMI";
```
Comment (RadxaYuntian) [inline arch/arm64/boot/dts/rockchip/overlays/rk3568-hdmi-disable.dts:10]:
```suggestion
		description = "Disable HDMI.";
```

File: arch/arm64/boot/dts/rockchip/overlays/rockchip-gpu-disable.dts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/dts-v1/;
+/plugin/;
+
+/ {
+	metadata {
+		title = "Disable GPU";
+		compatible = "rockchip,rk3308", "rockchip,rk3328", "rockchip,rk3399", "rockchip,rk3566", "rockchip,rk3568", "rockchip,rk3588";
+		category = "misc";
+		exclusive = "gpu";
+		description = "Disable GPU";
```
Comment (RadxaYuntian) [inline arch/arm64/boot/dts/rockchip/overlays/rockchip-gpu-disable.dts:10]:
```suggestion
		description = "Disable GPU.";
```

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]

File: arch/arm64/boot/dts/rockchip/overlays/rockchip-gpu-disable.dts [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+	metadata {
+		title = "Disable GPU";
+		compatible = "rockchip,rk3308", "rockchip,rk3328", "rockchip,rk3399", "rockchip,rk3566", "rockchip,rk3568", "rockchip,rk3588";
+		category = "misc";
+		exclusive = "gpu";
+		description = "Disable GPU.";
+	};
+};
+
+&gpu {
```
Comment (RadxaYuntian) [inline arch/arm64/boot/dts/rockchip/overlays/rockchip-gpu-disable.dts:14]: 你这个在3588上和mali的overlay互斥吧，有做判断么？

Comment (CodeChenL) [reply arch/arm64/boot/dts/rockchip/overlays/rockchip-gpu-disable.dts:14]: rk3588-mali-enable使用了相同的exclusive值 https://github.com/radxa-pkg/radxa-overlays/blob/d24f87e5ef988d457c2db3822940569f38f0039c/arch/arm64/boot/dts/rockchip/overlays/rk3588-mali-enable.dts#L9

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]
squash

**Review by RadxaYuntian — APPROVED** [review]

**Pull request merged**